### PR TITLE
Swaps out reporting document ids with document count

### PR DESCRIPTION
**Pull request by cdmo**

### CODE REVIEW

File: lib/tasks/incrementals.rake [hunk 1]
```diff
@@ -85,9 +85,9 @@ namespace :incrementals do
     end
 
     if ids.any?
-      deleted_ids = ids.join ','
+      formatted_count = ids.count.to_s.chars.reverse.each_slice(3).map(&:join).join(',').reverse
```
Comment (banukutlu) [inline lib/tasks/incrementals.rake:88]: 👀

**Review by banukutlu — APPROVED** [review]

File: lib/tasks/incrementals.rake (hunk 1 above)
Comment (banukutlu) [reply lib/tasks/incrementals.rake:88]: just to note here: this is to format `8888888` as `8,888,888`

Comment (cdmo) [reply lib/tasks/incrementals.rake:88]: could add a quick comment. I was actually kinda surprised that `Integer` didn't have a format option 🤷

Comment (banukutlu) [reply lib/tasks/incrementals.rake:88]: yup, same here

**Pull request merged**